src/Xunit.Combinatorial/CombinatorialMemberDataAttribute.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Andrew Arnott. All rights reserved.
```

Comment:
There seems to be a "Zero Width No-Break Space" here, which shows up as a line change. That shouldn't be there, right?